### PR TITLE
High-priority bug fixes for the June 15th demo

### PR DESCRIPTION
**Pull request by karpawich**
This PR fixes a number of high-priority bugs that @amyzlc requested I resolve in advance of an upcoming demo that she has with T+ID Lab partners on June 15th.

### CODE REVIEW

File: fetch/package-lock.json
Comment (steveand117) [inline fetch/package-lock.json:1]: I see that you have lockfileVersion 3. I think we ran into some issues if our node version was too high, which is why we were using Node v12 still. If you didn't run into them, then perhaps we can update?

**Review by steveand117 — COMMENTED** [review]
Looks good! The only thing that I was unsure about was the version of node you're using/changes to the package-lock.

File: fetch/package-lock.json
Comment (karpawich) [reply fetch/package-lock.json:1]: I'm running Node version v16.15.1.

What kinds of problems did y'all encounter? I did not encounter any that I'm aware of.

Comment (karpawich) [reply fetch/package-lock.json:1]: I'll Slack you my question.

**Pull request merged**